.travis.yml

```diff
@@ -17,7 +17,7 @@ after_success:
 notifications:
   slack:
     rooms:
-      - surveydatacollection:da6uVlmyrFhqINdm25bkLDiu
+      secure: "NcFdQQ5QbZu6naz8goI75PLKkKrlgkEV6td9yYZ4Wt13ZgYdT+rE5NtQiLQbPqvX56/Lphmm8sYm2VV/I92ibK5/Xbr0RGDQ3nl6kcmdwmHDSsPPVWYQvAtuzcvsxI0CGVQ+ek/pZYFBNOnsqHi4Dxfe1jdcVZROUSVao8xE0Hey63wDNj5UNA8S6qOBEyJSMOZ5tR6R5Xr83LteVGGuvNlPnOFu5ehZKZLPChWlyOcnHcdHZbHCiclDJn/fYV40EQ2NcU2uoN+HxLPKHUBAK/DYUkSOzVkHJdDN3rn9SVWvu/Z9rDHF+0jPVPEmNexV4QfE1oR5OVhj/CdPo5K1D1sVSfbVZlsIj93ThrARdm3meA5MNJdBcPF1o6ira8tf1xw8CcVXuj4hV6ywcin0pHvMMMIWQx7HqVo1WCX1lRyf2HnP+Qvu69kGJVPQ0HCosYlMI9P5woqerdtayuX0UlYiGr8/0a1vKnN78zxM6LnabmK9KPW/xGVcY5H43vJwYjvkdSib8EA5+EwDj3u0HR9LRtv0Wi7Yoa1sKT/ZL8dXobIWGJef/vX4M9PaTqx5dA+kyr8L5JcCEBTfMvOaW6Q5UDkec28NIMh4F4CAQt2avgoBW/fZqLmSleNNtRnhXaGhODgzOnRSyILxCMfrcxWpFCbWzWwIPOFx2IR3f0k="
```

Comment:
Are the quotes needed?

Reply:
generated by the travis cli, not required in yaml but won't cause problems.